build.py

```diff
@@ -319,16 +429,41 @@ def update(args: argparse.Namespace, env: dict[str, str]):
     if args.use_cuda:
         cuda_arch = 80
         cuda_compiler = str(args.cuda_home / "bin" / "nvcc")
-        command += [f"-DCMAKE_CUDA_COMPILER={cuda_compiler}",
-                    f"-DCMAKE_CUDA_ARCHITECTURES={cuda_arch}"]
+        command += [
+            f"-DCMAKE_CUDA_COMPILER={cuda_compiler}",
+            f"-DCMAKE_CUDA_ARCHITECTURES={cuda_arch}",
+        ]
 
     if args.android:
         command += [
             "-DCMAKE_TOOLCHAIN_FILE="
-            + str((args.android_ndk_path / "build" / "cmake" / "android.toolchain.cmake").resolve(strict=True)),
+            + str(
+                (
+                    args.android_ndk_path
+                    / "build"
+                    / "cmake"
+                    / "android.toolchain.cmake"
+                ).resolve(strict=True)
+            ),
             f"-DANDROID_PLATFORM=android-{args.android_api}",
             f"-DANDROID_ABI={args.android_abi}",
-            f"-DENABLE_PYTHON=OFF"
+            f"-DENABLE_PYTHON=OFF",
```

Comment:
```suggestion
            "-DENABLE_PYTHON=OFF",
```

Existing code, but this line doesn't actually need the `f`